app/data_models/relationship_store.py

```diff
@@ -89,7 +91,7 @@ def remove_all_relationships_for_list_item_id(self, list_item_id: str):
             self._is_dirty = True
 
     @staticmethod
-    def _build_map(relationships):
+    def _build_map(relationships: Union[dict, Union[dict, list]]) -> dict:
```

Comment:
relationships should be a mapping and return should be similar to the other store `dict[tuple[str, str], Relationship]`

Reply:
Ok, added, thanks.